### PR TITLE
Renamed getNanoTime to getElapsedTime

### PR DESCRIPTION
**Pull request by eshryane**
To be clear it's not an absolute value, just an offset.

### CODE REVIEW

**Comment by eshryane** [comment]
Waiting for integration tests to finish

File: whois-commons/src/test/java/net/ripe/db/whois/common/TestDateTimeProvider.java [hunk 1]
```diff
@@ -34,15 +34,11 @@ public ZonedDateTime getCurrentDateTimeUtc() {
         return getCurrentDateTime().atZone(ZoneOffset.UTC);
     }
 
-    public long getNanoTime() {
-        return DateTimeProvider.toEpochMilli(localDateTime);
+    public long getElapsedTime() {
+        return 100L;    // return a predicatable non-zero value that's testable
```
Comment (sbusk) [inline whois-commons/src/test/java/net/ripe/db/whois/common/TestDateTimeProvider.java]: predicatable -> predictable

File: whois-commons/src/main/java/net/ripe/db/whois/common/DateTimeProvider.java [hunk 2]
```diff
@@ -13,8 +13,8 @@
     LocalDateTime getCurrentDateTime();
     ZonedDateTime getCurrentDateTimeUtc();
 
-    /** returns System.nanoTime(), the high-res timer that counts 0 from JVM startup */
-    long getNanoTime();
+    /** returns elapsed time. N.B. not related to system time */
+    long getElapsedTime();
```
Comment (sbusk) [inline whois-commons/src/main/java/net/ripe/db/whois/common/DateTimeProvider.java]: Maybe call it getElapsedTimeNanos() so it's clear that the unit is nanons and not milliseconds..?

Comment (eshryane) [reply whois-commons/src/main/java/net/ripe/db/whois/common/DateTimeProvider.java]: I'd prefer not to include the unit precision in the method name, it makes it very verbose. I updated the comment to be more specific.

**Comment by eshryane** [comment]
Completed changes, waiting for integration tests to finish

**Comment by eshryane** [comment]
Integration tests have all passed

**Comment by eshryane** [comment]
If everyone is happy, I'd like to merge this change to include in Whois 1.95

**Pull request merged**